Vagrantfile

```diff
@@ -3,6 +3,10 @@
 
 # Automated creation of testing environments / binaries on misc. platforms
 
+$cpus = Integer(ENV.fetch('VMCPUS', '4'))  # create VMs with that many cpus
+$xdistn = Integer(ENV.fetch('XDISTN', '4'))  # dispatch tests to that many pytest workers
```

Comment:
this is a rather low value (also used in tox.ini as default) - uses little memory and causes relatively low load.